packages/editor/src/components/widgets/table/row/SelectRow.css

```diff
@@ -2,9 +2,16 @@
   cursor: pointer;
 }
 .selectable-row:where(:hover) {
-  background-color: var(--N50);
+  background-color: var(--A50);
 }
 
 .selectable-row:where([data-state='selected']) {
-  background-color: var(--N100);
+  background-color: var(--A400);
+  color: var(--background);
+}
+
+.selectable-row:where([data-state='selected']:last-child) {
+  background-color: var(--A400);
+  color: var(--background);
+  border-radius: var(--border-radius) !important;
```

Comment:
why is this needed???

Reply:
You are correct; it is no longer necessary. I encountered an issue where the border-radius from the table.css was not being applied. In an attempt to address this, I applied the following CSS. However, I have removed it, and everything is still works fine.